README.md

```diff
@@ -70,6 +70,13 @@ production:
   password: <%= ENV['circuitverse_DATABASE_PASSWORD'] %>
 ```
 
+- If you are facing difficulties installing RVM (which can be the    case with most users on ubuntu trying to install rvm) try using
```

Comment:
Hi, can the bracket message be trimmed to `(mostly on Ubuntu machine)` or something of that sort cause `trying to install rvm` seems a little repetitive. 
Also, in one line can you also explain how this is solving the problem.

Reply:
Ok I will do it